CMakePresets.json

```diff
@@ -0,0 +1,41 @@
+{
+  "version": 2,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 19,
+    "patch": 1
+  },
+  "configurePresets": [
+    {
+      "name": "WindowsEditor",
```

Comment:
does the name support slashes? if so, it would be interesting to do `<platform>/<config>/<target>`

For this particular preset, the configure doesnt need to specify a target nor a config. So you can name this "Windows" instead. If you prefer, you can name it "Windows-VS2019" since we may need a "Windows-VS2021" soon.